releasenotes/notes/adjust-symbolic-pulses-amp-angle-representation-f5c40007416cf938.yaml

```
@@ -0,0 +1,39 @@
+---
+other:
+  - |
+    Qiskit Terra 0.23.0 began phasing out support of complex parameters
+    in the Pulse module. Mainly, all library symbolic pulses were converted
+    from complex amplitude representation to a duo of real (float) parameters
+    (``amp``, ``angle``). To avoid problems, Qiskit Experiments adopted this convention.
+    
+    Changes were made to :class:`.FixedFrequencyTransmon` and :class:`.HalfAngleCal`
+    (see upgrade section). With the exception of :class:`.HalfAngleCal`, all 
+    library experiments should continue to function as they did before (even with 
+    complex ``amp``). When used with the :class:`.FixedFrequencyTransmon` library, 
+    :class:`.HalfAngleCal` will also continue working as before.
+    
+    Eventually, support for complex parameters will be dropped altogether, and it is
+    thus pending deprecation - including for saving and loading calibration data with
+    complex values.
+upgrade:
+  - |
+    The minimal required Qiskit-Terra version was bumped to 0.23 in order to use 
+    the (amp, angle) representation of library pulses in Qiskit Pulse.
```

If you want to be consistent with the next point:

```suggestion
    the (``amp``, ``angle``) representation of library pulses in Qiskit Pulse.
```